dipy/sims/voxel.py

```diff
@@ -997,6 +997,44 @@ def multi_tensor_msd(mf, mevals=None, tau=1 / (4 * np.pi ** 2)):
         msd += f * single_tensor_msd(mevals[j], tau=tau)
     return msd
 
+def ivim_signal(gtab, S0, f, D_star, D):
```

Comment:
This seems to duplicate functionality already implemented [in the ivim module](https://github.com/nipy/dipy/blob/master/dipy/reconst/ivim.py#L21). Could we reuse that instead?

Reply:
Yes, I have replicated this in `sims` so that I can use it to test the MIX module which is another PR. 
I can take it down from this PR I guess! Let me know :)

Reply:
Yes. Please remove this and use the implementation in the ivim module as needed.

Reply:
> Yes. Please remove this and use the implementation in the ivim module as needed.

Will do!